docs(sdk): Add Linear project description template (#17861)
getsentry/sentry-docs: develop-docs/sdk/getting-started/playbooks/coordination/managing-linear-projects.mdx

- **Initiative**: link to the appropriate initiative or create one (see [Initiatives](#initiatives) above); **MUST** be set before work begins
- **Lead**: a single owner (not a team)
- **Status**: set to the appropriate state (`Planned`, `In Progress`, etc.)
- **Target date**: realistic delivery date; **MUST** be set before the project moves to **In Progress**
- **Team**: the owning SDK team
- **Summary**: one sentence describing what this project ships and why it matters (separate field in Linear, just below the title)
- **Description**: use the structured template below; it should be detailed enough that a new team member can orient themselves without asking

You should also set:

- **Customers**: link relevant customer accounts or support tickets that motivated the work
- **Resources**: add relevant references (especially for larger initiatives) — design documents ([RFC](https://github.com/getsentry/rfcs), DACI, PRFAQ), Slack channels, related projects, and so on

> - **Milestones**: define intermediate delivery milestones for longer projects to track progress incrementally

Reviewer: ```suggestion
- **Milestones**: define intermediate delivery milestones to track progress incrementally for projects that last more than 2 weeks
```

- **Status**: set to the appropriate state (`Planned`, `In Progress`, etc.)
- **Target date**: realistic delivery date; **MUST** be set before the project moves to **In Progress**
- **Team**: the owning SDK team
- **Description**: use the structured template below; detailed enough that a new team member can orient themselves without asking
- **Resources**: add relevant references — design documents (RFC, DACI, PRFAQ), Slack channels, related projects

You **SHOULD** also set:

- **Customers**: link relevant customer accounts or support tickets that motivated the work
- **Milestones**: define intermediate delivery milestones for longer projects to track progress incrementally

A project without a lead, initiative, or target date **MUST NOT** be moved to **In Progress**.

##### Project Description Template

The description **SHOULD** follow this structure (copy and fill in):

```markdown
## Summary
<!-- One sentence: what this project ships and why it matters -->

## User-facing changes
<!-- What will developers/users be able to do differently after this ships? -->

> -

Author: ```suggestion

```

- **Status**: set to the appropriate state (`Planned`, `In Progress`, etc.)
- **Target date**: realistic delivery date; **MUST** be set before the project moves to **In Progress**
- **Team**: the owning SDK team
- **Summary**: one sentence describing what this project ships and why it matters (separate field in Linear, just below the title)

> - **Description**: use the structured template below; detailed enough that a new team member can orient themselves without asking

Reviewer 2: ```suggestion
- **Description**: use the structured template below; it should be detailed enough that a new team member can orient themselves without asking
```

You should also set:

- **Customers**: link relevant customer accounts or support tickets that motivated the work

> - **Resources**: add relevant references (especially for larger initiatives) — design documents ([RFC](https://github.com/getsentry/rfcs), DACI, PRFAQ), Slack channels, related projects

Reviewer 2: ```suggestion
- **Resources**: add relevant references (especially for larger initiatives) — design documents ([RFC](https://github.com/getsentry/rfcs), DACI, PRFAQ), Slack channels, related projects, and so on
```

> - **Milestones**: define intermediate delivery milestones for longer projects to track progress incrementally

Reviewer 3: Maybe also linking to existing/creating new initiatives if required?

Author: This was already there a bit further up :) I've expanded it a bit in https://github.com/getsentry/sentry-docs/pull/17861/changes/eae325237bc6881657e51c4dbe89b6964c6f3887

A project without a lead, initiative, or target date **MUST NOT** be moved to **In Progress**.

##### Project Description Template

Use this template as a starting point for the project description. For cross-SDK rollout projects where the initiative already covers the shared context (motivation, scope, outcomes), the per-SDK project description can reference the initiative and only document SDK-specific deviations (e.g., different scope, additional risks, platform-specific dependencies).

```markdown

> ## Why are we doing this?

Reviewer 3: Including customer signals/anecdotes is usually what helps me understand the issues the best. Could be a smaller comment/addition here.

Author: Good idea, added in https://github.com/getsentry/sentry-docs/pull/17861/changes/eae325237bc6881657e51c4dbe89b6964c6f3887 :)

<!-- 1–3 sentences on motivation — what we know so far and why we're kicking this off -->

## User-facing changes
<!-- What will developers/users be able to do differently after this ships? -->

## Expected outcomes
<!-- Be specific: "Reduce P0 support tickets for X", "Unblock platform Y adoption", "Increase metric Z" -->

## In scope
<!-- What will this project deliver? List the specific features, changes, or outcomes that are included -->

## Out of scope
<!-- Explicitly parking things here prevents scope creep and sets expectations -->

## Dependencies
<!-- Who or what must move before or alongside this? Flag blockers early -->

## Risks
<!-- What's most likely to go wrong? What's the mitigation? -->

> ## Definition of Done

Reviewer 3: Do we want to template this, or allow every project to define their own definition of done - e.g. closed issues, deployed, tested etc?

Reviewer 3: Looking at the section below it looks like there is already a section for project-specific ones?

Author: We already have general DoD criteria here: https://develop.sentry.dev/sdk/getting-started/standards/planning-scoping/#definition-of-done. The idea of the "Project-specific criteria" was to allow every project to define additional criteria :)

- **Status**: set to the appropriate state (`Planned`, `In Progress`, etc.)
- **Target date**: realistic delivery date; **MUST** be set before the project moves to **In Progress**
- **Team**: the owning SDK team
- **Description**: use the structured template below; detailed enough that a new team member can orient themselves without asking

You should also set:

- **Customers**: link relevant customer accounts or support tickets that motivated the work
- **Resources**: add relevant references (especially for larger initiatives) — design documents ([RFC](https://github.com/getsentry/rfcs), DACI, PRFAQ), Slack channels, related projects
- **Milestones**: define intermediate delivery milestones for longer projects to track progress incrementally

A project without a lead, initiative, or target date **MUST NOT** be moved to **In Progress**.

##### Project Description Template

Use this template as a starting point for the project description:

```markdown

> ## Summary

Reviewer 4: summary is a separate field in Linear, just below the title; think it's better to move this to the required attributes instead of project description template

Author: Addressed in fc2c90d27 — moved Summary to the required attributes list.

<!-- 1–3 sentences on motivation — what we know so far and why we're kicking this off -->

## Expected outcomes
<!-- Be specific: "Reduce P0 support tickets for X", "Unblock platform Y adoption", "Increase metric Z" -->

## In scope
<!-- What will this project deliver? List the specific features, changes, or outcomes that are included -->

## Out of scope
<!-- Explicitly parking things here prevents scope creep and sets expectations -->

## Dependencies
<!-- Who or what must move before or alongside this? Flag blockers early -->

## Risks
<!-- What's most likely to go wrong? What's the mitigation? -->

## Definition of Done

### Project-specific criteria
<!-- What must be true for this specific project to be done? -->
- [ ]

### Have we updated
<!-- Delete items that don't apply -->
- [ ] [Public docs](https://docs.sentry.io/)
- [ ] [Develop Docs](https://develop.sentry.dev)
- [ ] [Sentry Conventions](https://getsentry.github.io/sentry-conventions/)
- [ ] [Sentry CLI](https://github.com/getsentry/cli)
- [ ] [sentry-wizard](https://github.com/getsentry/sentry-wizard)
- [ ] Public SDK example projects
- [ ] In-product prompts/onboarding
- [ ] Internal adoption/usage dashboards

> ### Have you thought about doing a

Reviewer 4: this could also be automated ... once checked here ... create a Linear issue for it

Author: Definitely, we can have a look at this in a follow-up then 👍🏻

Verify closure against the [Definition of Done](/sdk/getting-started/standards/planning-scoping#definition-of-done) standard before setting the project to **Done**.

##### Artifact Checklist

Before closing, verify whether the project requires updates to any of the following. Not every item applies to every project — check the ones relevant to your change:

- [Sentry Public docs](https://docs.sentry.io/)
- [Sentry Develop Docs](https://develop.sentry.dev)
- [Sentry Conventions](https://getsentry.github.io/sentry-conventions/)
- [Sentry CLI](https://github.com/getsentry/cli)
- [sentry-wizard](https://github.com/getsentry/sentry-wizard)
- Public SDK samples
- In-product prompts/onboarding
- Internal adoption/usage dashboards

##### Communication Checklist

For user-facing features, consider whether any of the following are warranted:

- Write a [Sentry Product Blog](https://blog.sentry.io/) post
- Write a [Sentry Changelog](https://sentry.io/changelog/) entry

> - Write a [Sentry Engineering Blog](https://sentry.engineering/blog) post

Author: ```suggestion
- Write a [Sentry Engineering Blog](https://blog.sentry.io/engineering/) post
```

> - Write an [internal blog post](https://vanguard.getsentry.net/)

Author: Add S&T

Author: Also integrate into list above - split into sections "Project-specific" vs. "General"

Verify closure against the [Definition of Done](/sdk/getting-started/standards/planning-scoping#definition-of-done) standard before setting the project to **Done**.

> ##### Close-Out Steps

Author: ```suggestion
#### Close-Out Steps
```
